src/OCServo.h

```diff
@@ -0,0 +1,89 @@
+#ifndef OCSERVO_H
+#define OCSERVO_H
+
+#include <Arduino.h>
+#include <SoftwareSerial.h>
+
+#include "ROBSMotor.h"
+
+enum OCSMode {
+    OCS_SERVO,
+    OCS_MOTOR
+};
+
+enum OCSResponseLevel {
+    OCS_RESPOND_READ_ONLY,
+    OCS_RESPOND_ALL
+};
```

Comment:
- Using enum class provides better type safety and prevents implicit conversions, ensuring that enumerators don’t conflict with other values or enumerations in the global namespace.
- doc strings very useful add them to functions also
- UPER case is usually used for defines, use camel case

```suggestion
/**
 * @brief Enumeration for the operating mode of OCServo
 */
enum class OCSMode {
    Servo,
    Motor
};

/**
 * @brief Enumeration for the response level of OCServo
 */
enum class OCSResponseLevel {
    RespondReadOnly,
    RespondAll
};
```